docs/general/events.md

```diff
@@ -87,15 +67,15 @@ Recommendations on using attributes vs. body fields:
 
 * If the field should be comparable across every type of record, it should be an attribute.
 * If the field is specific to the event itself, then it should be a body field.
-* Unless the same `event.name` exists on two events, anything in two event bodies is not comparable to each other.
+* Unless two events share the same `EventName` property value, anything in two event bodies is not comparable to each other.
```

Comment:
I think we should reword the "two events" definition, as now it reads (to me) more specific than the previous trailing "two events"... Perhaps something like
(really just removing the "two" qualification as someone is likely to ask the question of what about if three, four, five events etc.)

"Unless the `EventName` property value matches, any fields or values within event bodies that share the same name or path should not be considered comparable."

With maybe a qualifier "Unless explicitly called out by the specific event definition" (or we can leave this out as edge cases (which they are).

And maybe a general follow up qualification about case-sensitivity (although I'm assuming that by keeping the above comments about `Event names are subject to the same rules as [attribute names](/docs/general/attribute-naming.md)` this "should" avoid this question... (Hopefully)

Reply:
I've updated the text, ptal